.travis.yml

```diff
@@ -26,3 +36,7 @@ script:
     - OMP_NUM_THREADS=1 tests/src/eigenwave3d
     - python tests/eigenwave3d.py default --output
     - python tests/eigenwave3d.py read
+    - python tests/eigenwave3d.py pluto --execute --nthreads=4
+    - OMP_NUM_THREADS=4 tests/src/eigenwave3d_pluto
+    - python tests/eigenwave3d.py pluto --output
+    - python tests/eigenwave3d.py read
```

Comment:
Yes, just add the script, not the results folder or pybench itself. Although you might want to add pybench to `requirements.txt`, just in case. You might also want to add a short entry to the README in tests how to do the auto-tuning.